manifests/c/calibre/calibre/portable/5.36.0.0/calibre.calibre.portable.locale.en-US.yaml

```diff
@@ -0,0 +1,33 @@
+# Created with YamlCreate.ps1 v2.0.6 $debug=QUSU.7-2-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: calibre.calibre.portable
+PackageVersion: 5.36.0.0
+PackageLocale: en-US
+Publisher: calibre-ebook.com
+PublisherUrl: https://github.com/kovidgoyal/calibre
+# PublisherSupportUrl: 
+# PrivacyUrl: 
+Author: kovidgoyal
+PackageName: calibre
```

Comment:
> Not entirely sure if this belongs here, but I ran into an issue where in a script I wrote Calibre was the only package I was not able to install via its name property, is there a chance portable's name could be changed as to not share the same name with the normal version ?

Any issues you have with a Package that is here you can create on the [Issues tab](https://github.com/microsoft/winget-pkgs/issues)
However since you commented on my PR its not a problem, and you're correct, this field is incorrect and causing matching issues

Reply:
Thank you very much for your work :)